CapstoneProject/Assets/Scripts/Core/Controller/Dialogue/DialogueManager.cs

```diff
@@ -197,33 +213,177 @@ private void BeginText()
     {
         displayTextObject.SetActive(true);
         displayTextObject.GetComponent<TextMeshProUGUI>().enabled = true;
-        StartCoroutine(TextScroll("This is test dialogue. Wa ba ba go bo"));
+        //scrollingCoroutine = TextScrollInput("This is test dialogue.");
+        //StartCoroutine(scrollingCoroutine);
 
+    }
+
+    private IEnumerator MainDialogueLoop()
+    {
+        dialogueIsplaying = true;
+        
+        IEnumerator textCoroutine = null;
         
+        UnityEngine.Debug.Log("Main dialogue loop entered");
+
+
+
+
+        foreach(string dialogue in loadedDialogue)
+        {
+            dialogueTicker++;
+            UnityEngine.Debug.Log("Looping");
+
+            if (gameState.relationshipStatus == 2)
+            {
+                StartCoroutine(SearchForState(textCoroutine, "playerkillrepeat", "break"));
+                dialogueSkipRepeat = true;
+            }
+            else if (gameState.relationshipStatus == 1)
+            {
+                StartCoroutine(SearchForState(textCoroutine, "playerkill", "playerdeathrepeat"));
+                dialogueSkipRepeat = true;
+            }
+            else if (gameState.recentDeath > 2)
```

Comment:
this appears to do the exact same things as the first if, so should we just || the first if? might just need a relationship != 1 as well